packages/valory/services/apy_estimation/service.yaml

```diff
@@ -22,3 +22,13 @@ models:
         - '0x2a651563c9d3af67ae0388a5c8f89b867038089e'
         - '0x2b4c76d0dc16be1c31d4c1dc53bf9b45987fc75c'
       ipfs_domain_name: /dns/registry.autonolas.tech/tcp/443/https
+---
+public_id: valory/ledger:0.1.0
+type: connection
+config:
+  ledger_apis:
+    ethereum:
+      address: https://goerli.infura.io/v3/1622a5f5b56a4e1f9bd9292db7da93b8
+      chain_id: 5
+      poa_chain: true
+      default_gas_price_strategy: eip1559
```

Comment:
The connection override LGTM, 
However we also need the setup_smart contract added
Here we need the overide for the smart contract address